cpu/msp430_common/startup.c

```diff
@@ -34,6 +34,10 @@ __attribute__((constructor)) static void startup(void)
 {
     board_init();
 
+#ifdef MODULE_DBGPIN
+    dbgpin_init();
+#endif
```

Comment:
For cortex-m this is called before `board_init()`. Shouldn't this be here the case as well?

Reply:
nope. The `board_init()` for the MSP430 platforms does reset all pin configurations, so putting the `dbgpin_init()` before `board_init()` would render `dbgpin()` effect less.